### PR TITLE
Auto-subscribe team & Gutenberg design for design issues

### LINKED ISSUE #297: Auto-@ groups for design issues
## Description

It came up in a discussion recently that issues with the label `aspect: design`, which are [almost exclusively in the frontend](https://github.com/issues?q=is%3Aopen+is%3Aissue+archived%3Afalse+repo%3AWordPress%2Fopenverse+repo%3AWordPress%2Fopenverse-api+repo%3AWordPress%2Fopenverse-frontend+repo%3AWordPress%2Fopenverse-catalog+repo%3AWordPress%2Fopenverse-infrastructure+sort%3Aupdated-desc+label%3A%22%F0%9F%96%BC%EF%B8%8F+aspect%3A+design%22+), could use input from the entire contributor/maintainer base along with other external groups. I'm not subscribed to the frontend repo, but I would love to be notified of `aspect: design` issues I should take a look at! 

It seems that there is a [`subscribe-to-label` GitHub action](https://github.com/bytecodealliance/subscribe-to-label-action) we could set up that would automatically @ certain groups when a label is added to an issue! Would folks be okay exploring this? I think we also floated the idea of @-ing the Gutenberg Designers group for these as well.

CC @WordPress/openverse-maintainers

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by AetherUnbound**
## Fixes

Fixes #297 by @AetherUnbound

## Description

This PR adds a new, synced, GitHub action which will automatically @ `WordPress/openverse` and `WordPress/gutenberg-design` for issues that get tagged with "🖼️ aspect: design".

We can always remove or adjust certain groups (or remove the auto-subscribe altogether) down the line if we don't find this helpful :slightly_smiling_face:

## Testing Instructions

Unfortunately due to the nature of GHA, I'm not sure how to test this without merging it first.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by dhruvkb — APPROVED** [review]
LGTM, nice find on the workflow!

File: .github/subscribe_to_label.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "WordPress/gutenberg-design": [
+    "\ud83d\uddbc\ufe0f aspect: design"
```
Comment (krysal) [inline .github/subscribe_to_label.json:3]: Are these the emojis? They don't render like that in macOS, I wonder if they will match the existing labels 😅

CleanShot 2022-10-11 at 11 21 48@2x

Comment (AetherUnbound) [reply .github/subscribe_to_label.json:3]: I pasted the emojis in there, and one of our JSON linting actions changed it to the unicode values 🤷🏼‍♀️

**Comment by AetherUnbound** [comment]
@WordPress/gutenberg-design - the Openverse project uses the "🖼️ aspect: design" label on issues and PRs that have to do with design questions. We've proposed here a way of auto @-ing the Gutenberg design group on said issues, in order to surface these on your feed and make it easier to keep in touch with Openverse design work without having to subscribe to the entire project. While we'd love to have your feedback on various design proposals/discussions, it's by no means obligatory 😄 

Our question to y'all is: **Would adding this as part of our workflow be constructive for collaboration, or would it be more of a nuisance/do more harm than good?**

**Comment by paaljoachim** [comment]
@AetherUnbound 
Hi Madison

For me personally it depends on time and interest when I respond to a Gutenberg design issue.

> Would adding this as part of our workflow be constructive for collaboration?

Yes. Please do. It is a good way to keep a tap on what is going on. I would then also have a chance to give feedback.

**Comment by jasmussen** [comment]
To echo Paal, I'd love to follow along and provide any help I can! Would be happy to try this!

**Review by zackkrida — APPROVED** [review]

**Pull request merged**